assets/js/components/Sectors.vue

```diff
@@ -237,18 +247,40 @@ export default Vue.extend({
       label.moveToBack();
 
       $this._labels
-      .data($this.getRoot().descendants().slice(1))
-        .style("display", (d) => { return d.data.enabled ? "inline" : "none"; })
-        .style("opacity", function(d) { return !d.data.enabled ? 0 : this.opacity; })
+       .data($this.getRoot().descendants().slice(1))
+       .on("click", function(d){
+            if (d.depth == 1 && d.data.enabled == true) {
+           $this.reset()
+          }
+        })
         .transition()
-        .duration(10)
+        .duration(400)
+        .style("display", (d) => { return d.data.enabled ? "block" : "none"; })
+        .style("opacity", function(d) { return !d.data.enabled ? 0 : this.opacity; })
```

Comment:
we might want to consider css transitions instead of its usual tweens. just switch classes and be done with it